scrapers/__init__.py

```diff
@@ -3,3 +3,4 @@
 from .victrola import scrape_victrola
 from .heart import scrape_heart
 from .bluebottle import scrape_bluebottle
+from .forty9p import scrape_49p
```

Comment:
probably because i couldn't figure out how to call the file if it started with a number